client/src/helpers/APICalls/winner.ts

```diff
@@ -0,0 +1,28 @@
+import { FetchOptions } from '../../interface/FetchOptions';
+import { Winner } from '../../interface/User';
+
+export const getWinnersByUser = async (): Promise<Winner[]> => {
+  const fetchOptions: FetchOptions = {
+    method: 'GET',
+    credentials: 'include',
+    headers: { 'Content-Type': 'application/json' },
+  };
+  return await fetch(`/winners`, fetchOptions)
+    .then((res) => res.json())
+    .catch(() => ({
+      error: { message: 'Unable to connect to server. Please try again' },
+    }));
+};
+
+export const getSomeWinners = async (num: number): Promise<Winner[]> => {
```

Comment:
I'm guessing we are getting a list of winners from all contests to have a separate winners page using this data?

Reply:
We can do whatever we want with these winners. According to the mock, we should probably render them on the profile page. But maybe we can add a section for previous winners for the logged in user and/or all previous winners. I wrote a route to get any number of winners to display on the front end.

Reply:
Also, since all the contests are being deleted post the deadline date you may need to refactor the profile page as we are fetching the completed contests under a separate tab.

Reply:
The contest won't be deleted on the deadline date. It will just be updated to be "active: false". A contest will be deleted only after the user chooses a winner. For that reason, we will still need all the contests as well as the winners here. I'll address this in my FE Choosing a winner PR.